hack/cass-stress-gen.py

```diff
@@ -79,4 +87,4 @@ def create_job_list(args):
     else:
       f = open(args.name + '.yaml', 'w')
       f.write('\n---\n'.join(manifests))
-      f.close
\ No newline at end of file
+      f.close
```

Comment:
?

Reply:
I have no idea... Perhaps `f.close()` is what is meant although python can be truly magic sometimes.